### PR TITLE
MF-291: Add yield pills to position list page

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

**Review by feraway — APPROVED** [review]

**Comment by feraway** [comment]
LGTM :)

File: src/config/constants/addresses.ts [hunk 1]
```diff
@@ -291,7 +291,8 @@ export const COINGECKO_IDS = {
   [NETWORKS.mumbai.chainId]: 'mumbai',
 };
 
-export const NETWORKS_FOR_MENU = [NETWORKS.optimism.chainId, NETWORKS.polygon.chainId];
+// export const NETWORKS_FOR_MENU = [NETWORKS.optimism.chainId, NETWORKS.polygon.chainId];
```
[the remark this answers is not in the document]
Comment (FiboApe) [reply src/config/constants/addresses.ts:294]: For now this version works only with polygon, untill we have new subgraphs deployed

File: src/config/constants/common.ts [hunk 2]
```diff
@@ -15,4 +15,9 @@ export type PositionVersions = PositionVersion2Type | PositionVersion3Type | Pos
 
 export const LATEST_VERSION: PositionVersions = POSITION_VERSION_4;
 
-export const POSITIONS_VERSIONS: PositionVersions[] = [POSITION_VERSION_2, POSITION_VERSION_3, POSITION_VERSION_4];
+// export const POSITIONS_VERSIONS: PositionVersions[] = [POSITION_VERSION_2, POSITION_VERSION_3, POSITION_VERSION_4];
```
[the remark this answers is not in the document]
Comment (FiboApe) [reply src/config/constants/common.ts:18]: same as before, for now only working with verison 4

File: src/home/current-positions/components/finished-position/index.tsx [hunk 3]
```diff
@@ -191,6 +196,12 @@ const ActivePosition = ({
 
   const isOldVersion = position.version === POSITION_VERSION_2;
 
+  const foundYieldFrom =
+    position.from.underlyingTokens[0] &&
```
[the remark this answers is not in the document]
Comment (FiboApe) [reply src/home/current-positions/components/finished-position/index.tsx:200]: We are certain because of types :)

**Pull request merged**